node-notes-cli/app.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+const myFile = './data.json';
+const dataJ = require('./data.json');
+const fs = require('fs');
+if (process.argv[2] === 'read') {
+  fs.readFile(myFile, 'utf8', (err, data) => {
+    if (err) throw err;
+    const parseStuff = dataJ.notes;
+    const countStuff = Object.keys(parseStuff).length;
+    for (var i = 1; i <= countStuff; i++) {
+      console.log(`${i}: ${parseStuff[i]}`);
+    }
+  });
+} else if (process.argv[2] === 'create') {
+  fs.readFile(myFile, 'utf8', (err, data) => {
```

Comment:
No need to read the file anymore, since you already brought the data in with `require()`